src/app/shared/axiosHelper.ts

```diff
@@ -22,9 +22,18 @@ interface RequestLog {
   duration?: number;
 }
 
+const ALLOWED_ENDPOINTS: string[] = [
```

Comment:
could name this is a bit more explicitly eg. `ALLOWED_RESPONSE_DATA_LOGGING_ENDPOINTS` ?